File: AtlasWeb/Core/AtlasWebSocketManager/Containers/AtlasActionContainer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Core.AtlasWebSocketManager.Containers
{
    public class AtlasActionContainer : AtlasWebSocketContainer, I_AtlasWebSocketContainer
    {
        public AtlasActionType actionType { get; set; }
        public bool isExecuted { get; set; }
        public string actionDefinition { get; set; }
        public string actionMethod { get; set; }

        public AtlasActionContainer() : base()
        {

        }

    }
}

File: AtlasWeb/Core/wwwroot/app/Fw/Models/AtlasActionContainer.ts
import { AtlasWebSocketContainer} from "./AtlasWebSocketContainer";

export class AtlasActionContainer extends AtlasWebSocketContainer {
    public actionType: AtlasActionType;
    public isExecuted: boolean;
    public actionDefinition: string;
    public actionMethod: string;
}

export enum AtlasActionType {

    CallMethod,
    NotifyUser,
    UpdateWorkList

}